infrastructure/team-dev/aws/us/routing/dns-public.tf

```diff
@@ -0,0 +1,21 @@
+resource "aws_route53_record" "public_apex" {
```

Comment:
@MattSurabian ownership is a bit odd here. I like creating all the DNS zones and certs in one central location (`infrastructure/routing`) to avoid all the whacky chicken-egg problems that arise trying to create in the project and delegate from outside... but it does mean that per-project workspaces need access to whatever account the DNS zones were created in. In theory it would be possible to mint the zones into the project itself regardless of the provider but I haven't really thought this through far enough yet to know if it makes sense.